scripts/setup_velma_os.sh

```diff
@@ -53,6 +63,12 @@ if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ] || [ -z "$4" ]; then
 	exit 1
 fi
 
+if [ $velma_hw -eq "1" ] && [ $velma_sim_gazebo -eq "1" ]; then
```

Comment:
czy te przełączniki na 100% się wykluczają? Jeśli tak to może da się tak zrobić, żeby się nie wykluczały?